robolectric/src/test/java/org/robolectric/shadows/ShadowNotificationBuilderTestBase.java

```diff
@@ -247,4 +248,42 @@ public void withBigPictureStyle() {
 
     assertThat(shadowOf(notification).getBigPicture().sameAs(bigPicture)).isTrue();
   }
+
+  @Test
+  public void build_whenAutoCancelNotSet_leaveAutoCancelAsFalse() {
+    Notification notification = builder.build();
+    assertThat(shadowOf(notification).isAutoCancel()).isFalse();
+  }
+
+  @Test
+  public void build_whenAutoCancelSet_setAutoCancelAsTrue() {
+    Notification notification = builder.setAutoCancel(true).build();
+    assertThat(shadowOf(notification).isAutoCancel()).isTrue();
+  }
+
+  @Test
+  @Config(minSdk = KITKAT_WATCH)
+  public void build_whenLocalOnlyNotSet_leaveLocalOnlyAsFalse() {
+    Notification notification = builder.build();
+    assertThat(shadowOf(notification).isLocalOnly()).isFalse();
+  }
+
+  @Test
+  @Config(minSdk = KITKAT_WATCH)
```

Comment:
This is supported from API 20 https://developer.android.com/reference/android/app/Notification.Builder#setLocalOnly(boolean)